crates/cli/src/subcommands/generate/csharp.rs

```diff
@@ -5,14 +5,12 @@ use std::ops::Deref;
 
 use convert_case::{Case, Casing};
 use spacetimedb_lib::sats::db::def::TableSchema;
-use spacetimedb_lib::sats::{
-    AlgebraicType, AlgebraicType::Builtin, AlgebraicTypeRef, ArrayType, BuiltinType, MapType, ProductType, SumType,
-};
+use spacetimedb_lib::sats::{AlgebraicType, AlgebraicTypeRef, ArrayType, BuiltinType, MapType, ProductType, SumType};
 use spacetimedb_lib::{ReducerDef, TableDesc};
 use spacetimedb_primitives::ColList;
 
```

Comment:
I'd like a comment at the top of this file with a link to the C# interfaces used by the code.

Like, "see SpacetimeDB/crates/bindings-csharp/Runtime for the attributes and types used here." Also BSATN.Runtime if that's used directly, but if it's imported by the other runtime crate that's fine.